pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -185,9 +195,32 @@ func (vm *windows) Configure() error {
 	if err := vm.transferFiles(); err != nil {
 		return errors.Wrap(err, "error transferring files to Windows VM")
 	}
+	if err := vm.ConfigureWindowsExporter(); err != nil {
+		return errors.Wrapf(err, "error configuring Windows exporter on the Windows VM %s", vm.ID())
+	}
+
 	return vm.runBootstrapper()
 }
 
+// Start Windows metrics exporter service, only if the file is present on the VM
+func (vm *windows) ConfigureWindowsExporter() error {
+	exists, err := vm.FileExists(windowsExporterPath)
```

Comment:
Given my previous comment we can remove this check given we are making this a required file.